sssom/parsers.py

```diff
@@ -260,11 +284,19 @@ def from_sssom_rdf(
                         k = "subject_id"
 
                     if isinstance(o, URIRef):
+                        v: Any
                         v = curie_from_uri(o, prefix_map)
                     else:
                         v = o.toPython()
-
                     if k:
+                        if (
+                            is_multivalued_slot(k)
+                            and v is not None
+                            and isinstance(v, str)
+                            and "|" in v
+                        ):
+                            # IF k is multivalued, then v = List[values]
+                            v = [s.strip() for s in v.split("|")]
                         mdict[k] = v
```

Comment:
Maybe refacror 292-299 and reuse in 214?

Reply:
Done